src/conf_mode/interfaces-bridge.py

```diff
@@ -123,6 +130,14 @@ def verify(bridge):
             if 'is_bond_member' in interface_config:
                 tmp = next(iter(interface_config['is_bond_member']))
                 raise ConfigError(error_msg + f'it is already a member of bond "{tmp}"!')
+            
+            if 'backup_port' in interface_config and 'backup_port_exist' not in interface_config:
+                tmp = interface_config['backup_port']
+                raise ConfigError(error_msg + f'Device {tmp} is not a bridge member and cannot be used as an alternate port!')
+            
+            if 'backup_port' in interface_config and 'backup_port_exist' in interface_config:
+                if not re.search('^((eth|lan)[0-9]+|(eno|ens|enp|enx).+)$', interface):
```

Comment:
This is redundant code - we have an interface name validator which should be used from the XML level. This prevents invalid input to pass down into the Python script.

The validator is named `interface-name` and also supports VLAN interfaces - those can not be used for a bridge backup port, but the validator can be adjusted to validate with and without vlans.

Reply:
done
But now I want to limit the use to the default Ethernet network card in the system, so I add a verifier based on this verifier.

Reply:
Please rename the validator to `interface-name-eth`

Reply:
done